pkg/assume/assume.go

```diff
@@ -524,6 +524,14 @@ func AssumeCommand(c *cli.Context) error {
 			}
 
 			clio.Successf("Exported credentials to ~/.aws/credentials file as %s successfully", profileName)
+
+			err := cfaws.ExportAccessTokenToCache(profile)
```

Comment:
Also not entirely sure if putting it inside `assumeFlags.Bool("export")` is good, open for suggestions.

Reply:
Hi @cedieio 

reading through the use case for this I believe that making this a flag of the`granted sso login` command would be more suitable

for example `granted sso login --use-plain-text-cache` 

For convenience we could update the login command to list sso start URLs from the config file if they exist

Reply:
hmm, moving the flag to granted is a valid point, but could we keep `assume` reading from `~/.aws/sso/cache` bit ?

This would reduce number of logins when authenticating with `aws sso login --session test-session` 

This would ease up the flow on our part since we're heavily using AWS login and having to login using assume again is confusing for users.

Reply:
re: @JoshuaWilkes I just remembered a use case we encountered the other day for keeping the flag on assume.

we were using images that mount the entire `~/.aws` volume, some tools use the `~/.aws/credentials` and some use the `~/.aws/sso/cache` depending on the implementation.

For this our entry point is just assume so I guess for our use case we'll need assume to still export the creds to plaintext.

Moving this to granted would add extra steps for users that need the functionality if they use the assume binary only.

Reply:
> re: @JoshuaWilkes I just remembered a use case we encountered the other day for keeping the flag on assume.
> 
> we were using images that mount the entire `~/.aws` volume, some tools use the `~/.aws/credentials` and some use the `~/.aws/sso/cache` depending on the implementation.
> 
> For this our entry point is just assume so I guess for our use case we'll need assume to still export the creds to plaintext.
> 
> Moving this to granted would add extra steps for users that need the functionality if they use the assume binary only.

Both Granted and Assume are actually bundled into the same binary `assumego` where granted is a symlink to `assumego`

`assume` is a shell script which calls `assumego` 

Not sure if this helps clarify your usecase

Reply:
Hi @JoshuaWilkes, thanks for getting back to me.

Just to clarify things if we move `--use-plain-text-cache` to `granted sso login` would it also work if we pass in `assume --use-plain-text-cache` ?

We're working with `assume` a lot and don't use `granted sso login` regularly as `assume` already does the stuff we need(logging in and setting cli AWS ENVs).

`assume` reading plain text from sso cache is also huge for us as it avoids duplicate logins when running `assume` if you have a valid sso access token in `~/.aws/sso/cache`

Cheers,
Cedric

Reply:
bumping this one @JoshuaWilkes

Reply:
Hi Cedric!
So sorry that the review has dragged out so long.
I've had another read over it this morning and I think the only change I would like to make is to add a seperate flag to assume instead of bundling into --export
If you can add the flag --export-sso-token which exports to plain text just as you have it in the assume command.
The assumer code looks good to me.
I think it is best to separate the flags so that the behaviour doesn't change for existing users and so that for most users we maintain the added benefit of having the tokens in secure storage.
usage would be assume <my-profile> --export --export-sso-token

Reply:
Thanks for the feedback @JoshuaWilkes, I've separated the export sso token to it's separate flag.

Cheers,
Cedric